src/gt4py/next/ffront/fbuiltins.py

```diff
@@ -175,7 +175,11 @@ def where(
 
 
 @builtin_function
-def astype(field: Field | gt4py_defs.ScalarT, type_: type, /) -> Field:
+def astype(
+    field: Field | gt4py_defs.ScalarT | Tuple[Field, ...],
+    type_: type,
+    /,
+) -> Field | Tuple[Field, ...]:
```

Comment:
since this function only returns a `NotImplementedError()`, shouldn't the return type here be `None`?

Reply:
I hear you but the type here seems to define the return type of the built-in function. If I replaced it by `None`, an error will be raised that the return type is "Illegal". Also, all built-in functions defined in this file have their proper return types.